docs/proposals/organizations.md

```diff
@@ -0,0 +1,125 @@
+# Organizations
+
+## Problems
+ 1.  We need to be able to allocate quota to a group of projects.
+ 2.  We need to be able to limit a particular project's usage of shared quota (think htb rate/ceil for tc qdisc classes).
+ 3.  We need to be able to manage groups across related projects, but not across the entire cluster.
+ 4.  We need to be able to limit how many projects a given User/Organization can have.  (limits provisioning)
```

Comment:
Would rephrase this as:

* Administrators want to allow self-service of users on the cluster, but the total allocated resources those self service users can get access to is limited (to prevent abuse / unfair use of resources).